low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -252,7 +252,7 @@ def _check_mismatch_count(self, device):
                         faulty_device = data.split(":")[0].rstrip()
                         if device == faulty_device:
                             self.alert_type = self.FAULT_RESOLVED
-                            self._alert_msg = "Mismatch_cnt found '0' for " + device
+                            self._alert_msg = "Data inconsistency fault detected in RAID protected disks, in %s RAID array is resolved." %device
```

Comment:
Sorry @shriya-deshmukh, can you add 'prior' to the statement,
"Prior data inconsistency fault detected in RAID protected disks, in %s RAID array is resolved."

Reply:
Done.